File: client/src/app/_models/sql-server-badge.ts
export class SqlServerBadge{
     serverId:string;
     name:string;
     serverAddress:string; 
     description :string;
     pageReadsCount :number = 0;
     requestCount :number = 0;
     pageLifetime :number = 0;
     connectedUsers:number = 0;
     isAlive:boolean = true;
}

File: src/ToolBox.Services.SQLMonitor/Domain/Models/SqlServerBadge.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ToolBox.Services.SQLMonitor.Domain.Models
{
    public class SqlServerBadge
    {
        public Guid ServerId { get; set; }
        public string Name { get; set; }
        public string ServerAddress { get; set; }
        public string Description { get; set; }
        public double PageReadsCount { get; set; }
        public double RequestCount { get; set; }
        public double PageLifetime { get; set; }
        public bool IsAlive { get; set; }
        public int ConnectedUsers { get; set; }
    }
}